src/modules/launchbox/enums/leaderboard.enum.ts

```diff
@@ -0,0 +1,16 @@
+export enum ChannelSlug {
+    NFT = "nft",
+    FARCASTER = "farcaster"
+}
+
+
+
+export enum FarcasterActions {
+    FOLLOW_CHAN = "follow_chan", // FIX: rename to channel_follow
+    CAST = "casts" // FIX: rename to channel_casts
```

Comment:
@noelukwa Fix ?

Reply:
oh yea.. just for semantic reasons, FarcasterActions.CHANNEL_CASTS instead of FarcasterActions.CAST